plugins/module_utils/network/iosxr/facts/acl_interfaces/acl_interfaces.py

```diff
@@ -44,6 +44,9 @@ def __init__(self, module, subspec="config", options="options"):
 
         self.generated_spec = utils.generate_dict(facts_argument_spec)
 
+    def get_config(self, connection):
+        return connection.get("show running-config interface")
```

Comment:
I'm assuming you're adding this function to make it easier to mock the result of `get_config`, but I am curious why you are calling `get` and not continuing to use `get_config` here

Reply:
yes

Reply:
+1 for changing this to `get_config`